website/templates/emails/new_public_project.html.mako

```diff
@@ -0,0 +1,33 @@
+## -*- coding: utf-8 -*-
+<%inherit file="notify_base.mako"/>
```

Comment:
@hmoco this isn't a style guide issue and simply my thoughts. I like to add whitespace around my comments and inside of these weird templates for readability e.g. `<% inherit file="..." />` and `# this is my comment` instead of `<%inherit file="..."/>` and `#this is my comment`